operators/install-kas-fleetshard.sh

```diff
@@ -20,4 +20,31 @@ ${KUBECTL} wait --timeout=90s \
     deployment/kas-fleetshard-operator \
     --namespace=${NAMESPACE}
 
+if [ -n "${SSO_TRUSTED_CA:-}" ] ; then
```

Comment:
If not explicitly exported in the .env file the `SSO_TRUSTED_CA` is not "available" here, I think it should be explicitly exported in the kas-installer.sh